script/GenKey.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# usage: ./GenKey.sh [<directory_name>]
+
```

Comment:
At the top here, I would do a `command` test to ensure that `openssl` is available or fail gracefully.  http://mywiki.wooledge.org/BashFAQ/081